module/documents/chat-message.mjs

```diff
@@ -652,6 +652,7 @@ export default class ChatMessage5e extends ChatMessage {
    * @protected
    */
   _enrichUsageEffects(html) {
+    if ( this.getFlag("dnd5e", "displayOnly") ) return;
```

Comment:
I think we're actually planning on taking out the display of effects for cards without activities, so rather than set the flag who don't you just tweak this function to remove the `else` block below (which should allow simplifying to move the effects getter out of the if statement.

```javascript
    if ( this.getFlag("dnd5e", "messageType") !== "usage" ) return;
    const item = this.getAssociatedItem();
    const effects = this.getFlag("dnd5e", "use.effects")
      ?.map(id => item?.effects.get(id))
      .filter(e => e && (game.user.isGM || (e.transfer && (this.author.id === game.user.id))));
```